### ISSUE TITLE
New dracula organization

### ISSUE THREAD

**Issue by nesl247** [yl0g0ly/dracula-ui#3]
Hi @yl0g0ly,

Dracula just went through a large reorganization in order to better support the growing number of editors, shells, and more that we support. We would love it if you would be willing to migrate this to the new organization here at `https://github.com/dracula`. You would still be the maintainer the theme, it just makes it easier to manage all of the different dracula related themes if they are under the same roof.

Looking forward to what you think.
- The Dracula Theme Team

**Comment by ichi0g0y** [comment]
Hi @nesl247,

It's so good reorganisation!

I will try migration to dracula theme project.
But, I'm not understood steps for migration.
Moreover, my English is not good X(

Can you tell me steps for migration?

**Comment by nesl247** [comment]
Hi @yl0g0ly,

GitHub actually has a nice guide on how to do the migrations here https://help.github.com/articles/transferring-a-repository/. Unfortunately as we are finding out, APM (Atom Package Manager) might be an issue. We're still trying to migrate our own atom syntax without having to unpublish and republish in Atom.

Please wait until we've figured out how to handle Atom before you migrate. We don't want to add any unneeded complicates to anyone. I'll reply back here when we've figured it out.

Thanks again!

**Comment by ichi0g0y** [comment]
Hi @nesl247,

thanks for reply.

I wait until it.
Migration of that time github repository also performs.
Waiting for your return.

thanks again.

**Comment by nesl247** [comment]
Hi @yl0g0ly. We ended up deciding against hosting any UI themes under the organization for the moment. The reason is that UI themes are a lot of work, and editors change frequently enough to make it a potential nightmare.

However, I have added a link to your theme in the README for our atom syntax theme. I haven't tried out your theme yet, but the screenshot looks amazing. I'm looking forward to seeing how things progress in the future.

We're working on our official policy for UI themes though, so we might get back in touch in the future. Thanks again, and good luck. Hope to see lots of users coming your way! 😄
